frontend/src/config.ts

```diff
@@ -54,3 +54,15 @@ export const SENTRY_RELEASE = process.env.REACT_APP_SENTRY_RELEASE as string
 export const SENTRY_ENVIRONMENT =
   (process.env.REACT_APP_SENTRY_ENVIRONMENT as string) || 'development'
 export const INFO_BANNER = process.env.REACT_APP_INFO_BANNER as string
+
+// Feature Launch Announcements
+// If `isActive` is false, the modal will not proc for ANY user
+export const ANNOUNCEMENT = {
+  isActive: true,
```

Comment:
```suggestion
  isActive: process.env.REACT_APP_ANNOUNCEMENT_ACTIVE || false,
```

Maybe make this an env var?

Reply:
Good point, thanks~